Jenkinsfile

```diff
@@ -52,7 +52,7 @@ SUPPORTED_STAGES = [
 ]
 
 // Supported VM Images
-SLAVE_IMAGE = 'gcr.io/endpoints-jenkins/debian-9:0.12'
+SLAVE_IMAGE = 'gcr.io/endpoints-jenkins/debian-9:0.13'
```

Comment:
Why does this need to be updated?

Reply:
Seems the environment installed in script/linux-install-software should be included in this image, which will be used in jenkins.

Reply:
Good question.  My impression is jenkins slave image is specified here.  But actually, it is in the jenkins endpoint config.  Well, to be safe, also update it here.